### ISSUE TITLE
 some issues occurs when running  phasebook with example data. 

### ISSUE THREAD

**Issue by chenwuanhui** [phasebook/phasebook#8]
Hi,
 some issues occurs when running  phasebook with example data. And in the 3.cluster folder,  no ref.polished.fa file generated, i do not know how ti solve this , Look forward to your reply about this, thank you.
![微信图片_20211109144849](https://user-images.githubusercontent.com/93968816/140876359-8b12d4ae-c72a-4fdd-bb82-875d4757f8e3.jpg)
![微信图片_20211109144856](https://user-images.githubusercontent.com/93968816/140876372-05b4b9f5-a82f-4867-b5d2-a1c2f0cd1000.jpg)

**Comment by HaploKit** [comment]
Hi, could you add `--rm_tmp False` when running the example and show me the files in `ls -rt 3.cluster/c1/` ? thanks.

**Comment by chenwuanhui** [comment]
![微信图片_20211117180648](https://user-images.githubusercontent.com/93968816/142181822-31313f1c-ffbb-4a3e-bc6c-6d4d03a25993.jpg)
![微信图片_20211117180659](https://user-images.githubusercontent.com/93968816/142181854-5fe233e8-63a9-4963-92da-f4035c1341d6.jpg)

Hi, thank you so much for your response, i have add --rm_tmp False in command line, same files were generated in file folder 3.cluster/c1/ , but 1.ref.polished.fa is empty,  so the error was not solved. Looking  forward to your response.

**Comment by HaploKit** [comment]
It seems something wrong when running 'racon'. Could you please show me the information in `1.log` file ?

**Comment by chenwuanhui** [comment]
Thank you so much for your response， I have reinstall racon, and final contigs were generated, the size is 611Kb, seem not completely right.
![微信图片_20211118101251](https://user-images.githubusercontent.com/93968816/142338856-556e64eb-92fd-4883-8815-28004fcac3ed.png)

**Comment by HaploKit** [comment]
It should work now.

**Comment by chenwuanhui** [comment]
![c33f3807c1447118d6d158188a426b9](https://user-images.githubusercontent.com/93968816/142382413-8c48ac00-e2ea-4e90-a7eb-7c92acac5e68.png)
but now something wrong after I reinstall longshot and whatshap.

**Comment by HaploKit** [comment]
Do not have to reinstall longshot and whatshap. There is something incompatible if using different version of whatshap.

**Comment by chenwuanhui** [comment]
Ok thank you,I will check it. But why the final contigs is 661Kb, Someone obtain this file is 800Kb?

**Comment by chenwuanhui** [comment]
The 1.log file showed that longshot and whatshap were not performed successful, so I reinstall them.

**Comment by chenwuanhui** [comment]
Hi, finally, phasebooks can run correctly. And the final congtigs is 1.1Mb， but I wonder if the final contigs contain  both hap1 and hap2 sequence? , And how to separate them？ I am anticipating your reply.

**Comment by HaploKit** [comment]
Hi, yes, it contains haplotype-specific contigs from both haplotypes. But additional data (e.g. Hi-C, parental data) is necessary if one wants to separate them further. phasebook does not support this yet, which is left to future research.

**Comment by chenwuanhui** [comment]
Ok, thank you.

**Comment by chenwuanhui** [comment]
Hi, one more question, how could i know if whatshap  run correctly, i can not find any command line in 3.cluster/*.log about this tools?

**Comment by HaploKit** [comment]
You could check, for example, if there is a correct`1.phased.vcf.gz` file (generated by whatshap) in the `3.cluster/c1/` folder, but this needs to use the parameter `--rm_tmp False`. Or, you could simply check if there are folders like `hapUnknown` or `hap1` or/and `hap2` in the  `3.cluster/c1/` , because if whatshap fails, no further analysis such as phasing is performed.

**Comment by chenwuanhui** [comment]
Yes, there are 1.phased.vcf.gz in the 3.cluster/c1/ folder, thank you.

**Comment by chenwuanhui** [comment]
Hi, my data were divided into 1734 clusters, and 1642 clusters were assembled into *hap1.supereads.fa and *hap2.supereads.fa within one day, but the other 100 clusters can not finished in 5 days, anything wrong with it ? or there are any way to improve the speeds? Thank you.
My comman line: nohup python phasebook.py -i 9_nanopore.fasta -t 50 -p ont -g large -x --rm_tmp False &
